recipes/newrelic/infrastructure/awslinux.yml

```diff
@@ -7,6 +7,11 @@ description: New Relic install recipe for the Infrastructure agent
 repository: https://github.com/newrelic/infrastructure-agent
 
 installTargets:
+  - type: host
+    os: linux
+    platform: amazon
+    platformFamily: rhel
+    platformVersion: "2018.03"
```

Comment:
@amolero-nr the amazonlinux_1 file didnt specify a platformVersion, hence cover all of them. Is it safe/good to make it more specific here? Perhaps we know the other versions are not supported by the infra agent?

Reply:
Thank you @Julien4218! - I thought to stick to that version and not prior ones after reading [this end-of-life/maintenance support notes from AWS](https://aws.amazon.com/blogs/aws/update-on-amazon-linux-ami-end-of-life/) but will remove that line for now. I have not tested if prior versions are supported  or not by the infra agent.